lib/hardware/modules/reg/iob_reg/iob_reg.py

```diff
@@ -34,16 +33,12 @@ def setup(py_params_dict):
             {
                 "name": "clk_en_rst",
                 "type": "slave",
-                "port_prefix": "",
-                "wire_prefix": "",
                 "descr": "Clock, enable, and reset",
                 "signals": [],
             },
             {
                 "name": "io",
                 "type": "master",
-                "port_prefix": "",
-                "wire_prefix": "",
                 "descr": "Input and output",
                 "signals": [
                     {
```

Comment:
separate these signals in different ports. putting them in the same group gives issues 

only create groups if you are sure the signals always connect together ; otherwise separate them